lib/src/androidTest/java/com/google/android/material/composethemeadapter/MdcThemeTest.kt

```diff
@@ -146,16 +155,58 @@ class MdcThemeTest<T : AppCompatActivity>(activityClass: Class<T>) {
                 assertEquals(2.13f, blurRadius)
             }
 
+            typography.h3.run {
+                assertEquals(sansSerif.fontFamily, fontFamily)
+                assertEquals(sansSerif.weight, fontWeight)
+            }
+
+            typography.h4.run {
+                assertEquals(sansSerifLight.fontFamily, fontFamily)
+                assertEquals(sansSerifLight.weight, fontWeight)
+            }
+
+            typography.h5.run {
+                assertEquals(sansSerifBlack.fontFamily, fontFamily)
+                assertEquals(sansSerifBlack.weight, fontWeight)
+            }
+
+            typography.h6.run {
+                assertEquals(serif.fontFamily, fontFamily)
+                assertEquals(serif.weight, fontWeight)
+            }
+
             typography.body1.run {
                 assertTextUnitEquals(16.26.sp, fontSize, density)
                 assertTextUnitEquals(0.005.em, letterSpacing, density)
                 assertEquals(rubik400, fontFamily)
                 assertNull(shadow)
             }
 
+            typography.body2.run {
+                assertEquals(cursive.fontFamily, fontFamily)
+                assertEquals(cursive.weight, fontWeight)
+            }
+
+            typography.subtitle1.run {
+                assertEquals(monospace.fontFamily, fontFamily)
+                assertEquals(monospace.weight, fontWeight)
+            }
+
+            typography.subtitle2.run {
+                assertEquals(GenericFontFamily::class, fontFamily!!::class)
```

Comment:
Not sure on this assertion. It's not really testing anything. Can we check `GenericFontFamily.name`?